README.md

```diff
@@ -59,23 +61,17 @@ const processes = await run(
 From a range of numbers, divide them by 4, remove any greater than 12, then start collecting those one by one:
 
 ```js
-function* range(from, to) { 
-  for (let i = from; i <= to; i++) yield i  
+function* range(from, to) {
+  for (let i = from; i <= to; i++) yield i
 }
 
-const arr = run(
-  range(40, 60)
-, map(d => d / 4)
-, filter(d => d < 12)
-, reduce([])
-)
+const arr = run(range(40, 60), map(d => d / 4), filter(d => d < 12), reduce([]))
```

Comment:
It would be nice to have some better variable names here rather than just 1 character ones.

Reply:
You might be right, but this is also not the point of the pr.

I agree with the pr. The moment I saw the first code snippet my eyes burned.